shell/common/shell.h

```diff
@@ -106,7 +106,10 @@ class Shell final : public PlatformView::Delegate,
   fml::WeakPtr<Engine> weak_engine_;          // to be shared across threads
   fml::WeakPtr<Rasterizer> weak_rasterizer_;  // to be shared across threads
   fml::WeakPtr<PlatformView>
-      weak_platform_view_;  // to be shared across threads
+      weak_platform_view_;                 // to be shared across threads
+  int raster_cache_max_bytes_user_value_;  // The user-requested value for the
```

Comment:
std::optional<size_t> instead of magic values please.

Reply:
Going to move this to rasterizer as a bool flag per @jason-simmons suggestion instead